demo-certs/create_certs.sh

```diff
@@ -99,7 +106,9 @@ keytool -import -trustcacerts -noprompt -alias root -file $DIR/root-cert.pem -ke
 keytool -import -trustcacerts -noprompt -alias ca -file $DIR/ca-cert.pem -keystore $DIR/$HONO_TRUST_STORE -storepass $HONO_TRUST_STORE_PWD
 
 echo ""
-echo "writing CA public key into properties file"
+echo "extracting trust anchor information from CA cert"
+CA_SUBJECT=$(openssl x509 -in $DIR/ca-cert.pem -noout -subject -nameopt RFC2253 | sed s/^subject=//)
+sed -i "s%\(trusted-ca\.subject-dn=\).*\$%\1${CA_SUBJECT}%" $DIR/trust-anchor.properties
```

Comment:
why don't we simply overwrite or create the file here, e.g. using something like
```
echo "trusted-ca.subject-dn=$CA_SUBJECT" > $DIR/trust-anchor.properties
echo "trusted-ca.public-key=$PK" >> $DIR/trust-anchor.properties
```
looks a little easier to understand than the (highly sophisticated) `sed` command for replacing the value ...

Reply:
Done.